app/src/main/java/com/example/coinage/fragments/profile/EditInfoFragment.java

```diff
@@ -0,0 +1,104 @@
+package com.example.coinage.fragments.profile;
+
+import android.content.Context;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.TextView;
+
+import com.example.coinage.R;
+import com.parse.ParseUser;
+
+public class EditInfoFragment extends Fragment {
+    public static final String TAG = "EditInfoFragment";
+
+    private TextView tvNameEditDisplay;
+    private TextView tvNameEdit;
+    private ConstraintLayout clEmailEdit;
+    private ConstraintLayout clPasswordEdit;
+
+    public EditInfoFragment() {
+        // Required empty public constructor
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_edit_info, container, false);
+    }
+
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        tvNameEditDisplay = view.findViewById(R.id.tvNameEditDisplay);
+        tvNameEditDisplay.setText(ParseUser.getCurrentUser().getUsername());
+
+        tvNameEdit = view.findViewById(R.id.tvNameEdit);
+        tvNameEdit.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Log.i(TAG, "change name");
+                goEditName();
+            }
+        });
+
+        clEmailEdit = view.findViewById(R.id.clEmailEdit);
+        clEmailEdit.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Log.i(TAG, "change email");
+                goEditEmail();
+            }
+        });
+
+        clPasswordEdit = view.findViewById(R.id.clPasswordEdit);
+        clPasswordEdit.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Log.i(TAG, "change password");
+                goEditPassword();
+            }
+        });
+
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        // minimize keyboard input once returned to this fragment
+        InputMethodManager imm =(InputMethodManager) getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
```

Comment:
style - It's almost always best to avoid acronyms in variable names. I would say exceptions are when - 
1. the variable name is going to be RIDICULOUSLY long without abbreviating
2. It is an acronym you are  confident everyone reading will pick up immediately 
example: we use FB or OC in a lot of variable names, that's ok cause every coder in every language at Meta knows what they mean immediately. 

in oculus code, we would write out inputMethodManager as the variable name here